src/main/java/org/fluentd/logger/sender/ExponentialDelayReconnector.java

```diff
@@ -1,20 +1,18 @@
 package org.fluentd.logger.sender;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import java.util.LinkedList;
 
 /**
- * Calcurate exponential delay for reconnecting
+ * Calculates exponential delay for reconnecting. The start delay is 50ms and exponentially grows to max 60 seconds in
+ * function of the number of connection errors.
  */
 public class ExponentialDelayReconnector implements Reconnector {
 
-    private double wait = 0.5;
+    private double waitMillis = 50; // Start wait is 50ms
```

Comment:
Initial interval 50ms sounds a bit too short and the original initial value was supposed to be 500ms. Why don't you change it to 500ms?